conans/client/command.py

```diff
@@ -1222,7 +1228,24 @@ def upload(self, *args):
 
         args = parser.parse_args(*args)
 
-        if args.query and args.package:
+        try:
```

Comment:
This `try/except/else` is exactly the same code as the one in `def get(...)` function, do you think it is worth putting it all in a function and call it from both places?

Reply:
Probably, but they are not exactly. It would feel a bit obscure I think

Reply:
Ok, it was just a suggestion 👍